src/app/MD/page.jsx

```diff
@@ -8,6 +8,49 @@ import Search from "../components/search";
 import Link from "next/link";
 import Switch from "@mui/material/Switch";
 
+const SunIcon = ({ className = "" }) => (
```

Comment:
Can you create a file called icons in the components and just import it from there?